samples/bot-release-management/nodejs/helpers/devOpsHelper.js

```diff
@@ -28,7 +28,6 @@ class DevOpsHelper {
             AssignedToProfileImage: isAssignedToPresent ? await graphHelper.GetProfilePictureByUserPrincipalNameAsync(DevOpsHelper.GetEmailOrName(workItem.resource.fields[Constant.AssignedTo], false)) : "",
             CreatedByName: DevOpsHelper.GetEmailOrName(workItem.resource.fields[Constant.CreatedByKey], true),
             CreatedByProfileImage: image,
-            StakeholderTeam: DevOpsHelper.ValidateMails(workItem.resource.fields[Constant.StakeHolderTeamKey].split(',')),
```

Comment:
Why did you remove email validation part?

Reply:
In feilds object there was some properties, and the property **StakeHolderTeamKey** is not exist and its throwing error while splitting the value using split(',')

Email is exists in this property **CreatedByKey**.

So that line of code which i have removed is deprecated.